pkg/controller/jenkins/configuration/user/seedjobs/seedjobs.go

```diff
@@ -297,13 +313,27 @@ func (s *SeedJobs) isRecreatePodNeeded(jenkins v1alpha2.Jenkins) bool {
 	return false
 }
 
+func (s *SeedJobs) ensureAgentExecutorsAmount() (requeue bool, err error) {
+	// Check if the amount of executors is valid
+	if s.Configuration.Jenkins.Spec.Master.SeedJobAgentExecutors < minimalSeedJobAgentExecutors {
+		s.Configuration.Jenkins.Spec.Master.SeedJobAgentExecutors = minimalSeedJobAgentExecutors
+	}
+
+	err = s.Client.Update(context.TODO(), s.Configuration.Jenkins)
+	if err != nil {
+		return true, err
+	}
+
+	return false, nil
+}
```

Comment:
remove and add validation here https://github.com/jenkinsci/kubernetes-operator/blob/74048433223f215a48d45be586c0f27148ac3da4/pkg/controller/jenkins/configuration/user/validate.go#L10